pallets/pool-system/src/impls.rs

```diff
@@ -521,6 +522,14 @@ mod benchmarks_utils {
 				FUNDS.into(),
 			));
 
+			// Fund pool account
+			let pool_account = PoolLocator { pool_id }.into_account_truncating();
+			frame_support::assert_ok!(T::Tokens::mint_into(
+				POOL_CURRENCY,
+				&pool_account,
+				POOL_ACCOUNT_BALANCE.into()
+			));
```

Comment:
Just curious, why is this minting now needed?

Reply:
This was taken from the integration tests when I was debugging `BalanceTooLow` errors. So it might not be necessary but I believe it still is.

When we process a redemption of tranche tokens, the pool account needs to have native balance which is exchanged for these tranche tokens. 

In contrast, we don't need to fund the pool account with tranche tokens as these are minted when processing an investment.